dotcom-rendering/src/components/FlexibleSpecial.tsx

```diff
@@ -29,34 +90,43 @@ export const OneCardLayout = ({
 	showAge?: boolean;
 	absoluteServerTimes: boolean;
 }) => {
-	if (!cards[0]) return null;
+	const card = cards[0];
+	if (!card) return null;
 
+	const {
+		headlineSize,
+		headlineSizeOnMobile,
+		headlineSizeOnTablet,
+		imagePositionOnDesktop,
+		imagePositionOnMobile,
+		supportingContentAlignment,
+	} = determineCardProperties(
+		card.boostLevel,
+		card?.supportingContent?.length ?? 0,
+	);
 	return (
 		<UL padBottom={true}>
 			<LI padSides={true}>
 				<FrontCard
-					trail={cards[0]}
+					trail={card}
 					containerPalette={containerPalette}
 					containerType="flexible/special"
 					showAge={showAge}
 					absoluteServerTimes={absoluteServerTimes}
-					headlineSize="large"
-					headlineSizeOnMobile="medium"
-					imagePositionOnDesktop="right"
-					imagePositionOnMobile="top"
+					headlineSize={headlineSize}
+					headlineSizeOnMobile={headlineSizeOnMobile}
+					headlineSizeOnTablet={headlineSizeOnTablet}
+					imagePositionOnDesktop={imagePositionOnDesktop}
+					imagePositionOnMobile={imagePositionOnMobile}
 					imageSize="jumbo"
-					trailText={cards[0].trailText}
-					supportingContent={cards[0].supportingContent}
-					supportingContentAlignment={
-						cards[0].supportingContent &&
-						cards[0].supportingContent.length > 3
-							? 'horizontal'
-							: 'vertical'
-					}
+					trailText={card.trailText}
+					supportingContent={card.supportingContent}
+					supportingContentAlignment={supportingContentAlignment}
 					imageLoading={imageLoading}
 					aspectRatio="5:4"
-					kickerText={cards[0].kickerText}
-					showLivePlayable={cards[0].showLivePlayable}
+					kickerText={card.kickerText}
+					showLivePlayable={card.showLivePlayable}
+					boostedFontSizes={true}
```

Comment:
I've switched tact here and decided to pass through boostedFontSizes from the layout rather than an isSplash prop. My thinking for this is 
- we decide the rest of boost styling here
- we will have boosted cards that aren't splash who will want to access this prop